ci-operator/step-registry/mco/conf/OWNERS

```diff
@@ -0,0 +1,16 @@
+approvers:
+  - Xia-Zhao-rh
+  - kuiwang02
+  - bandrade
+  - jianzhangbjz
+  - sregidor
+  - rioliu-rh 
+  - aleskandro
+reviewers:
+  - Xia-Zhao-rh
+  - kuiwang02
+  - bandrade
+  - jianzhangbjz
+  - sregidor
```

Comment:
^

Reply:
I've changed the wrong user name. Thank you!